### PR TITLE
[FIX] Remove duplicated slashes from any URL that opens embedded in a WebView

### LINKED ISSUE #1111: [BUG] Admin page not loading
I just tried to open the admin page on our server with 0.60.2 in Build 132, but it just shows a spinning wheel and then a blank page.

### LINKED ISSUE #1118: [BUG] Sidebar doesn't update all values on logout
When logging out from one server and switching automatically to another server following params weren't updated:
 - Server Name (retaining old servers title)
 - Profile picture (set to default svg, neither updating to the current servers image nor retaining the old servers image)
 - Online status (offline)
So more or less the whole top elements except the username are not updated correctly in the sidebar.

### PR DESCRIPTION
**Pull request by rafaelks**
@RocketChat/ios

Closes #1111 
Closes #1118

### CODE REVIEW

File: Rocket.Chat/Extensions/URLExtension.swift [hunk 1]
```diff
@@ -81,4 +81,14 @@ extension URL {
         return newURL
     }
 
+    mutating func removingDuplicatedSlashes() -> URL {
```
Comment (cardoso) [inline Rocket.Chat/Extensions/URLExtension.swift]: Can you write tests for this @rafaelks? Does it work with https:// ?

Comment (rafaelks) [reply Rocket.Chat/Extensions/URLExtension.swift]: Done! :-)

Comment (cardoso) [reply Rocket.Chat/Extensions/URLExtension.swift]: Sorry for not catching this before, but does this method need to be mutating? If it must be mutating then we should rename it to "removeDuplicatedSlashes".

**Comment by soundstorm** [comment]
@rafaelks but this will break some URLs, right? 😢
e.g. https://www.aliexpress.com/item//32798469203.html (ok) and https://www.aliexpress.com/item/32798469203.html (404)

The main issue was the server url containing a trailing slash, so can't we just apply a stripper of the trailing slash if there is any?

**Comment by cardoso** [comment]
Then we should just advise the admin to remove the trailing slash? @rafaelks @soundstorm

**Comment by rafaelks** [comment]
@soundstorm @cardoso Yeah... I guess we can leave the app without this change for now. Thanks guys!

**Comment by soundstorm** [comment]
@cardoso I logged out, entered the url again without trailing slash but it fetched the //admin again (?)

**Comment by rafaelks** [comment]
Let's merge this one, for the better.

**Comment by rafaelks** [comment]
@cardoso This is ready to review again.

**Pull request merged**